Mathlib/CategoryTheory/Subobject/MonoOver.lean

```diff
@@ -219,6 +220,17 @@ theorem pullback_obj_arrow (f : X ⟶ Y) (g : MonoOver Y) :
 
 end Pullback
 
+section IsPullback
+
+/-- The pullback of `S` along `f` is isomorphic to `T` given the corresponding pullback square. -/
```

Comment:
Could you make the docstring more specific? (We should be able to understand most of what this definition is about by reading the docstring.)